content/python/concepts/built-in-functions/terms/filter/filter.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: 'filter()'
+Description: 'Returns a filter object that applies a function to each item in an iterable and returns the values that are True.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Strings'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+The `filter()` function returns a filter object that contains values filtered from an iterable where a function returns `True` for a given item's value.
+
+## Syntax
+
+```psuedo
+filter(func_name, iterable)
+```
+
+The `func_name` is the function to be applied to the iterable and can be a lambda function or the name of any defined function. The `iterable` contains the items, such as a list, the function will act on.
+
+## Example 1
+
+Use `filter()` to return all the odd values in the following list:
```

Comment:
```suggestion
The following example uses `filter()` to return all the odd values in the `nums` list:
```

For Docs entries, they should have a voice/tone that is declarative and not imperative.